app/js/app/controllers/MyBoardsPageControllers.js

```diff
@@ -115,15 +116,15 @@ define([], function() {
 					}
 				}
 				// Front-end filters
-				$scope.search = {
-					completion: '',
-					title: '',
-					subjects: '',
-					levels: '',
-					createdBy: '',
-					formattedCreationDate: '',
-					formattedLastVisitedDate: ''
-				}
+				$scope.exactMatch = {
+					completion: undefined,
+					createdBy: undefined
+				};
+				window.pm = $scope.partialMatch = {
```

Comment:
This is funky!
(But seems ok: https://stackoverflow.com/questions/1758576/multiple-left-hand-assignment-with-javascript)